core/src/main/java/jenkins/install/SetupWizard.java

```diff
@@ -113,6 +115,18 @@ public SetupWizard(Jenkins j) throws IOException {
      */
     public HttpResponse doCompleteInstall(StaplerRequest req, StaplerResponse rsp) throws IOException, ServletException {
         Jenkins j = Jenkins.getActiveInstance();
+        User u = j.getUser("admin");
+        // JENKINS-33572 - without creating a new 'admin' user, auth key erroneously remained
+        if(u != null && u.getProperty(AuthenticationKey.class) != null) {
+            // There must be a better way of removing things...
+            Iterator<Map.Entry<Descriptor<UserProperty>,UserProperty>> entries = u.getProperties().entrySet().iterator();
+            while(entries.hasNext()) {
+                Map.Entry<?, ?> entry = entries.next();
+                if(entry.getValue() instanceof AuthenticationKey) {
+                    entries.remove();
+                }
+            }
+        }
```

Comment:
I suppose you want to save the user after it. Otherwise the change won't be persisted